src/main/resources/nl/tudelft/ti2206/group9/util/saveGame.json

```diff
@@ -1,12 +1,12 @@
 {
 	"playername": "Mathias",
-	"coins": 42,
+	"coins": "42",
 	"settings": {
 			"soundEnabled": "true",
 			"otherSetting": "otherValue"
 	},
 	"highscore": {
-		"score": 2560.0,
-		"coins": 39
+		"score": "2560.0",
+		"coins": "39"
```

Comment:
Can I ask why integers and doubles are now saved as Strings? This seems a bit redundant.

<logic,review